src/geometry/minkowski.md

```diff
@@ -0,0 +1,97 @@
+<!--?title Minkowski sum of convex polygons -->
+# Minkowski sum of convex polygons
+
+## Definition
+Consider two sets $A$ and $B$ of points on a plane. Minkowski sum $A + B$ is defined as $\\{a + b| a \in A, b \in B\\}$.
+Here we will consider the case when $A$ and $B$ consist of convex polygons $P$ and $Q$ with their interiors.
+It turns out that the sum of convex polygons $P$ and $Q$ is a convex polygon with at most $|P| + |Q|$ vertices.
```

Comment:
Can you make it a little bit more clear that with convex polygon you always mean the vertices of the polygon. $P$ is the list of vertices of the polygon, and not the polygon itself.